README.md

```diff
@@ -224,6 +224,14 @@ Use `<INSTANCE>:<FILENAME>` to specify a source or target inside an instance.
 #### `limactl edit`
 `limactl edit <INSTANCE>`: edit the instance
 
+#### `limactl volume`
+
```

Comment:
Needs an explanation of "volume". (filesystem, mount point, size, ...)

Probably it should be also noted that this is unrelated to `nerdctl volume` volumes.
(So, `limactl storage` or `limactl disk` might be less confusing name?)

Reply:
> Needs an explanation of "volume". (filesystem, mount point, size, ...)

Yes, but not necessarily in the `README.md`. The other commands also include only the most common option and an abbreviated description. The question is where else it should go; ideally we would have some proper documentation, but who is going to write it?

> Probably it should be also noted that this is unrelated to nerdctl volume volumes.
> (So, limactl storage or limactl disk might be less confusing name?)

I don't know; "volume" was the first choice for this functionality for me too, because that's what it is.

But I understand the risk of confusion; just like people get confused between nerdctl namespaces and kubernetes namespaces.

I'm still fine with `volume` but would prefer `disk` over `storage`.

Reply:
I'd be ok with changing the naming to `disk`. I agree that `volume` would be better in isolation but I think it's likely this overlap will confuse a few people. I'll update in the next commit.

Reply:
I know we have already all agreed on `disk`, but I only now noticed that this means we now have both a `disk` and a `disks` key in `lima.yaml`, which is confusing.

if we are sticking with `disk` for the concept (I'm still fine with using `volume`), then maybe we should at least change the key in `lima.yaml` to `dataDisks`, to separate it from `disk` which really should have been `diskSize`.

@AkihiroSuda What do you think?

Reply:
What about `additionalDisks`?